extension/src/openvic-extension/classes/GFXButtonStateTexture.cpp

```diff
@@ -114,11 +110,11 @@ Error GFXButtonStateTexture::generate_state_image(
 		return { std::max(colour.r - 0.3f, 0.0f), std::max(colour.g - 0.3f, 0.0f), std::max(colour.b - 0.3f, 0.0f), colour.a };
 	};
 
-	const auto colour_func =
-		button_state == HOVER ? hover_colour :
-		button_state == PRESSED ? pressed_colour :
-		button_state == DISABLED ? disabled_colour :
-		selected_colour;
+	const auto colour_func = button_state == HOVER ? hover_colour
+		: button_state == PRESSED				   ? pressed_colour
+		: button_state == DISABLED
+		? disabled_colour
+		: selected_colour;
```

Comment:
- Using tab characters to align multiple lines together only works consistently when you're using the same number of characters starting from the same offset, otherwise it breaks when using an interface with a different tab display size (like it's breaking here for me in Github).
- Why are the first two `?`s kept on the same line but the third one is moved to a newline?

Reply:
Because clang-format has literally no idea what is going on after the first two ternary operators.